File: geek-university-algoritmo-e-logica-de-programacao/java/exercicio_16.java
import java.util.Scanner;

public class exercicio_16 {

    public static void main(String[] args) {
        int numero, a, b;
        Scanner teclado = new Scanner(System.in);

        System.out.print("Informe um numero: ");
        numero = teclado.nextInt();

        if (numero == 0) {
            System.out.println("O valor " + numero + " eh neutro.");
        } else if (numero > 0) {
            a = numero;
            System.out.println("O valor " + a + " eh positivo.");
        } else {
            b = numero;
            System.out.println("O valor " + b + " eh negativo.");
        }

        teclado.close();
    }

}

File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_16.c
#include <stdio.h>

int main() {
    int numero, a, b;

    printf("Informe um numero: ");
    scanf("%d", &numero);

    if (numero == 0) {
        printf("O valor %d eh neutro.", numero);
    } else if (numero > 0) {
        a = numero;
        printf("O valor %d eh positivo.", a);
    } else {
        b = numero;
        printf("O valor %d eh negativo.", b);
    }
}
